packages/agreement-process/src/services/agreementContractBuilder.ts

```diff
@@ -190,7 +198,6 @@ const getPdfPayload = async (
   eservice: EService,
   consumer: Tenant,
   producer: Tenant,
-  seed: UpdateAgreementSeed,
```

Comment:
Removing the `seed` parameter from `getPdfPayload` and using the `agreement`. In all cases where we call this function, we do it after the agreement was already updated with the seed - I find passing both `agreement` and  `UpdateAgreementSeed` confusing. In general, in a further PR, I was thinking about removing this concept of `UpdateAgreementSeed`, since where we use it we just create the seed object and then spread it right away to update the agreement - see for example: https://github.com/pagopa/interop-be-monorepo/blob/0f08caa7aa66e26bffc068f04d72150c151cebe3/packages/agreement-process/src/services/agreementService.ts#L1011